src/Runtime/OMTensorList.inc

```diff
@@ -68,32 +64,29 @@ struct OMTensorList {
 
   int64_t _size; // Number of elements in _omts.
 
-  int64_t _owning; // indicates whether the OMTensorList owns the pointer to
-                   // OMTensor array or not. OMTensorList struct will release
-                   // the memory space referred to by '_omts' upon destruction
-                   // if and only if it owns it.
+  bool _shallow; // indicates whether we want to perform a shallow destroy or
+                 // regular destroy when calling the explicit
+                 // omTensorListDestroyShallow or
+                 //  omTensorListDestroy method.
```

Comment:
Comment is wrong. C APIs don't use `_shallow`. It's only for the C++ constructor/destructor API (which isn't currently externalized yet).